components/Header.vue

```diff
@@ -201,7 +201,7 @@ export default class extends Vue {
     width: 149px;
   }
 
-  #getStartedInput {
+  # {
```

Comment:
unintentional?

Reply:
Oh this is actually intentional! Was removing it as it could potentially cause style override issues. This header component is actually not used anymore.

Reply:
Interesting. I've never seen a CSS selector like that before. What does it do? Select everything with IDs?